openshift/README.md

```diff
@@ -0,0 +1,40 @@
+
+**CD**: [ArgoCD resource](https://argocd-shared.apps.silver.devops.gov.bc.ca/applications?proj=&sync=&health=&namespace=&cluster=&labels=&search=platform-registry) is will compare the current deployment configration and the record we have in[tenant-gitops-platform-registry](https://github.com/bcgov-c/tenant-gitops-platform-registry/tree/main/platform-registry-app) and do the deploy the application accordingly.
+
+### Environment has different usage
+
+- [Dev](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-dev): Testing env for feature branch
+- [Test](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-test): Testing env for main branch
+- [Prod](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-prod): Production environment for user.
+
+#### Usage of different env's pipeline
+
+The pricinple for all environment is the same but the way how to trigger/use the pielines for each env are slightly different.
+
+##### Dev
+
+We are using s2i strategy for regsitry web and api, because it is used for testing feature branch, we will then target the source to the feature branch that you want to test in the BuildConfig in tools namespaces under for both web([platsrv-registry-web-dev-build](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-tools/buildconfigs/platsrv-registry-web-dev-build/yaml) ) and api([platsrv-registry-api-dev-build](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-tools/buildconfigs/platsrv-registry-api-dev-build/yaml) ) under `spec.source.git.ref`(ie. update your branch name into that field). **DON'T FORGET TO SAVE YOUR CHANGE.**
+
+Then you will need to manually trigger a new build in registry repo github action for both web([DEV Build and Tag - WEB](https://github.com/bcgov/platform-services-registry/actions/workflows/dev-web.yml)) and api([DEV Build and tag - API](https://github.com/bcgov/platform-services-registry/actions/workflows/dev-api.yml)) builds by clicking Run workflow on the right side of the page. (Normally you don't have to select branch, default is master branch unless you are testing your new github action branch).
+
+Wait untill those images been build and tagged and commited the image hash tag to GitOps repository(all automated process).
+
+Auto Sync policy in argocd [Dev instance](https://argocd-shared.apps.silver.devops.gov.bc.ca/applications/platform-registry-dev) has enabled, so argoCD will do a auto deployment once it detect that its `out of sync`. (if not, click refresh, because argocd don't scan resource on second base.) Once the current sync status becomes **Synced** again, your feature branch should have been deployed into Dev env.
```

Comment:
```suggestion
Auto Sync policy in argocd [Dev instance](https://argocd-shared.apps.silver.devops.gov.bc.ca/applications/platform-registry-dev) has been enabled, so argoCD will do a auto deployment once it detects that it's `out of sync`. (if not, click refresh, because argocd don't scan resource on second base.) Once the current sync status becomes **Synced** again, your feature branch should have been deployed into Dev env.
```